Warsaw_Tram/Extensions/UIViewController.swift

```diff
@@ -1,37 +1,33 @@
 //
-//  AlertHelper.swift
+//  UIViewController.swift
 //  Warsaw_Tram
 //
-//  Created by Małgorzata Dziubich on 25/06/16.
+//  Created by Małgorzata Dziubich on 26/06/16.
 //  Copyright © 2016 Małgorzata Dziubich. All rights reserved.
 //
 
 import UIKit
 
-protocol AlertHelperProtocol {
-    func showAlert(title: String, message: String, okAction: UIAlertAction, cancelAction: UIAlertAction, presentingViewController: UIViewController)
-    func showAlert(title: String, message: String, okAction: UIAlertAction, presentingViewController: UIViewController)
-}
+extension UIViewController {
```

Comment:
It is not what I mean. Extending object like this is anti-pattern. Go back to protocol `AlertHelperProtocol` and extend it only for `UIViewController` like this:
```extend AlertHelperProtocol where Self: UIViewController```